python/ray/serve/schema.py

```diff
@@ -59,206 +74,40 @@ def _route_prefix_format(cls, v):
-        default=DEFAULT.VALUE,
-        description=(
-            "Config specifying autoscaling "
-            "parameters for the deployment's number of replicas. "
-            "If null, the deployment won't autoscale its number of "
-            "replicas; the number of replicas will be fixed at "
-            "num_replicas."
-        ),
-    )
-    graceful_shutdown_wait_loop_s: float = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Duration that deployment replicas will wait until there "
-            "is no more work to be done before shutting down. Uses a "
-            "default if null."
-        ),
-        ge=0,
-    )
-    graceful_shutdown_timeout_s: float = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Serve controller waits for this duration before "
-            "forcefully killing the replica for shutdown. Uses a "
-            "default if null."
-        ),
-        ge=0,
-    )
-    health_check_period_s: float = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Frequency at which the controller will health check "
-            "replicas. Uses a default if null."
-        ),
-        gt=0,
-    )
-    health_check_timeout_s: float = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Timeout that the controller will wait for a response "
-            "from the replica's health check before marking it "
-            "unhealthy. Uses a default if null."
-        ),
-        gt=0,
-    )
-    ray_actor_options: RayActorOptionsSchema = Field(
-        default=DEFAULT.VALUE, description="Options set for each replica actor."
-    )
-
-    placement_group_bundles: List[Dict[str, float]] = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Define a set of placement group bundles to be "
-            "scheduled *for each replica* of this deployment. The replica actor will "
-            "be scheduled in the first bundle provided, so the resources specified in "
-            "`ray_actor_options` must be a subset of the first bundle's resources. All "
-            "actors and tasks created by the replica actor will be scheduled in the "
-            "placement group by default (`placement_group_capture_child_tasks` is set "
-            "to True)."
-        ),
-    )
-
-    placement_group_strategy: str = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "Strategy to use for the replica placement group "
-            "specified via `placement_group_bundles`. Defaults to `PACK`."
-        ),
-    )
 
-    max_replicas_per_node: int = Field(
-        default=DEFAULT.VALUE,
-        description=(
-            "[EXPERIMENTAL] The max number of deployment replicas can "
-            "run on a single node. Valid values are None (no limitation) "
-            "or an integer in the range of [1, 100]. "
-            "Defaults to no limitation."
-        ),
+    # Override default values
```

Comment:
why do we need to use `DEFAULT.VALUE` here instead of `None`?

Reply:
See above, we no longer need to override the default values in the schema